lavalink/Client.py

```diff
@@ -123,18 +114,34 @@ def unregister_hook(self, func):
         """ Updates a player's state when a payload with opcode ``playerUpdate`` is received. """
         guild_id = int(data['guildId'])
 
-        if guild_id in self.players:
-            player = self.players.get(guild_id)
-            player.position = data['state'].get('position', 0)
-            player.position_timestamp = data['state']['time']
+        for node in self.nodes:
+            if guild_id in node.players:
+                player = node.players.get(guild_id)
+                player.position = data['state'].get('position', 0)
+                player.position_timestamp = data['state']['time']

```

Comment:
Perhaps consider breaking the for loop to prevent further (needless) iterations

Reply:
done